doc/include/api_embedc.inc

```diff
@@ -263,42 +194,33 @@ be [const Tcl_Interp*]. In that case the argument in question is
 [comment ---------------------------------------------------------------------]
 [call [cmd ::critcl::cproc] [arg name] [arg arguments] [arg resulttype]]
 
-This variant of [cmd critcl::cproc] assumes that the functionality to
-connect to is implemented by the C function [arg name] which has the
-signature described by the [arg arguments] and [arg resulttype].
+As documnented below, but used when the C function named [arg name] already
+exists.
 
-[para] It creates only the shim performing the conversions required by
-arguments and result.
 
 [comment ---------------------------------------------------------------------]
 [call [cmd ::critcl::cinit] [arg text] [arg externals]]
 
-This command compiles the C code in [arg text] and [arg externals].
-
-[para] Both arguments have access to all definitions created by the
-previously listed commands, regardless of their placement in the
-[file .critcl] file relative to this command.
-See section [sectref {Runtime Behaviour}] for more details.
-
-[para] The C code in [arg text] is put into the body of the
-initialization function of the shared library backing the
-[file .critcl] file, and is executed when this library is loaded into
-the interpreter.
+Appends the C code in [arg text] and [arg externals], but only after all the
+other fragments appended by the previously-listed commands regardless of their
+placement in the [vset critcl_script] relative to this command.  Thus, all
+theri content is visible.  See also [sectref {Runtime Behaviour}].
 
-[para] This code has access to a variable [var interp] of type
-[const Tcl_Interp*] referencing the Tcl interpreter currently being
-initialized.
+[para] The C code in [arg text] is placed into the body of the initialization
+function of the shared library backing the [vset critcl_script], and is
+executed when this library is loaded into the interpreter.  It has access to
+the variable [var {Tcl_Interp* interp}] referencing the Tcl interpreter currently
+being initialized.
 
-[para] The code in [arg externals] on the other hand is placed outside
-and just before the initialization function, making this is a good
-place for any external symbols required by initialization function
-which should not be accessible by any other parts of the C code.
+[para] [arg externals] is placed outside and just before the initialization
+function, making it a good place for any external symbols required by
+initialization function, but which should not be accessible by any other parts
+of the C code.
 
-[para] Multiple invokations of this command are allowed, and a later
-call has access to the information of all preceding ones.
+[para] Calls to this command are cumulative.
```

Comment:
`[vset cumulative]` ?

Reply:
Done with commit 29380521d0873ea1dcc5f8cdff6742b20764473f (Branch `documentation` in my repo, not seen by the PR).